server/src/map.c

```diff
@@ -0,0 +1,90 @@
+/*
+** EPITECH PROJECT, 2024
+** Project
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#include "map.h"
+#include "egg.h"
+
+static void generate_stones(map_t *map, int x, int y)
+{
+    for (int i = 0; i < (map->height * map->width) / 10; i++) {
```

Comment:
this for loop can be extracted to remove code duplicates

Reply:
No because if i do that i won't know which size_t i need to increase